archetypes/Pools/PoolsTable/index.tsx

```diff
@@ -716,45 +718,55 @@ const UpOrDown: React.FC<{
     showNextRebalance: boolean;
     currency?: boolean;
     deltaDenotation: DeltaEnum;
-}> = ({
-    oldValue,
-    newValue,
-    deltaDenotation,
-    tooltipMetric,
-    showNextRebalance,
-    poolTicker,
-    tokenMetricSide,
-    currency = true,
-}) => {
-    const value = useMemo(
-        () =>
-            deltaDenotation === DeltaEnum.Numeric ? newValue - oldValue : calcPercentageDifference(newValue, oldValue),
-        [deltaDenotation, oldValue, newValue],
-    );
-    const approxValue = Math.abs(parseFloat(value.toFixed(3)));
-    return (
-        <UpOrDownTip
-            metric={tooltipMetric}
-            valueText={
+}> = React.forwardRef(
+    (
+        {
+            oldValue,
+            newValue,
+            deltaDenotation,
+            tooltipMetric,
+            showNextRebalance,
+            poolTicker,
+            tokenMetricSide,
+            currency = true,
+        },
+        _ref,
+    ) => {
+        const value = useMemo(
+            () =>
                 deltaDenotation === DeltaEnum.Numeric
-                    ? currency
-                        ? toApproxCurrency(value).replace('-', '')
-                        : approxValue.toString()
-                    : `${approxValue}%`
-            }
-            side={tokenMetricSide}
-            value={value}
-            currency={currency}
-            poolTicker={poolTicker}
-            showNextRebalance={showNextRebalance}
-        >
-            <UpOrDownInner
-                oldValue={oldValue}
-                newValue={newValue}
-                deltaDenotation={deltaDenotation}
-                currency={'USD'}
-                showCurrencyTicker={false}
-            />
-        </UpOrDownTip>
-    );
-};
+                    ? newValue - oldValue
+                    : calcPercentageDifference(newValue, oldValue),
+            [deltaDenotation, oldValue, newValue],
+        );
+        const approxValue = Math.abs(parseFloat(value.toFixed(3)));
+        return (
+            <UpOrDownWithTooltipTip
+                metric={tooltipMetric}
+                valueText={
+                    deltaDenotation === DeltaEnum.Numeric
+                        ? currency
+                            ? toApproxCurrency(value).replace('-', '')
+                            : approxValue.toString()
+                        : `${approxValue}%`
+                }
+                side={tokenMetricSide}
+                value={value}
+                currency={currency}
+                poolTicker={poolTicker}
+                showNextRebalance={showNextRebalance}
+            >
+                {/* Fixes ref error with antd tooltip */}
+                <div>
```

Comment:
I was looking into this warning earlier this week, tbh I couldn't quite figure out. It makes sense to use forward ref but the warning goes away just by adding this div or fragment.

Reply:
yeah I think the we would achieve the same thing by using forward ref on UpOrDown which we could do instead but is not needed for any case other than this

Reply:
merging for now and we can change later